cedar-drt/fuzz/fuzz_targets/eval-type-directed.rs

```diff
@@ -15,7 +15,6 @@
  */
 
 #![no_main]
-use cedar_drt::utils::expr_to_est;
```

Comment:
might be able to remove this utility function `expr_to_est` entirely

Reply:
Good catch, it's removed now.